internal/distro/rhel8/imagetype.go

```diff
@@ -413,8 +413,39 @@ func (t *imageType) PackageSetsNew(bp blueprint.Blueprint, options distro.ImageO
 		logrus.Errorf("Initializing the manifest failed for %s (%s/%s): %v", t.Name(), t.arch.distro.Name(), t.arch.Name(), err)
 		return nil
 	}
+	return resolvePackageSetChains(manifest.GetPackageSetChains())
+}
+
+// Runs resolvePackageNames() on each package set's Include and Exclude list
+// and replaces package names.
+func resolvePackageSetChains(chains map[string][]rpmmd.PackageSet) map[string][]rpmmd.PackageSet {
+	pkgSetChains := make(map[string][]rpmmd.PackageSet)
+	for name, chain := range chains {
+		cc := make([]rpmmd.PackageSet, len(chain))
+		for idx := range chain {
+			cc[idx] = rpmmd.PackageSet{
+				Include:      resolvePackageNames(chain[idx].Include),
+				Exclude:      resolvePackageNames(chain[idx].Exclude),
+				Repositories: chain[idx].Repositories,
+			}
+		}
+		pkgSetChains[name] = cc
+	}
+	return pkgSetChains
+}
 
-	return manifest.GetPackageSetChains()
+// Resolve packages to their distro-specific name. This function is a temporary
+// workaround to the issue of having packages specified outside of distros (in
+// internal/manifest/os.go), which should be distro agnostic. In the future,
+// this should be handled more generally.
+func resolvePackageNames(packages []string) []string {
```

Comment:
Good point.  I was thinking ahead to the feature-based one, but it's better to have the name describe what it's doing, not what it might be doing in the future.

Reply:
Done!